clang/include/clang/CIR/Dialect/Builder/CIRBaseBuilder.h

```diff
@@ -438,13 +438,15 @@ class CIRBaseBuilderTy : public mlir::OpBuilder {
   }
 
   mlir::Value createAdd(mlir::Value lhs, mlir::Value rhs, bool hasNUW = false,
-                        bool hasNSW = false) {
+                        bool hasNSW = false, bool saturated = false) {
```

Comment:
How about updating `createSub`? Can saturation be used with NSW/NUW too? Should those be updated?

Reply:
Sure, will do it with neon SISD sub as an driving test case